src/components/GeoSearchControls/GeoSearchControls.tsx

```diff
@@ -0,0 +1,110 @@
+/** @jsx h */
+
+import { h, Fragment } from 'preact';
+import cx from 'classnames';
+import Template from '../Template/Template';
+import GeoSearchButton from './GeoSearchButton';
+import GeoSearchToggle from './GeoSearchToggle';
+import {
+  GeoSearchCSSClasses,
+  GeoSearchTemplates,
+} from '../../widgets/geo-search/geo-search';
+import { ComponentCSSClasses } from '../../types';
+import { PreparedTemplateProps } from '../../lib/utils/prepareTemplateProps';
+
+type Props = {
+  cssClasses: ComponentCSSClasses<GeoSearchCSSClasses>;
+  enableRefine: boolean;
+  enableRefineControl: boolean;
+  enableClearMapRefinement: boolean;
+  isRefineOnMapMove: boolean;
+  isRefinedWithMap: boolean;
+  hasMapMoveSinceLastRefine: boolean;
+  onRefineToggle(event: Event): void;
+  onRefineClick(event: MouseEvent): void;
+  onClearClick(event: MouseEvent): void;
+  templateProps: PreparedTemplateProps<GeoSearchTemplates>;
+};
+
+const GeoSearchControls = ({
+  cssClasses,
+  enableRefine,
+  enableRefineControl,
+  enableClearMapRefinement,
+  isRefineOnMapMove,
+  isRefinedWithMap,
+  hasMapMoveSinceLastRefine,
+  onRefineToggle,
+  onRefineClick,
+  onClearClick,
+  templateProps,
+}: Props) => (
+  <Fragment>
```

Comment:
why do we need a Fragment? it seems everything is already wrapped with a div.

Reply:
wrapped a component in a fragment because of https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18912 / https://github.com/microsoft/TypeScript/issues/21699